highlighters/regexp/regexp-highlighter.zsh

```diff
@@ -0,0 +1,56 @@
+# -------------------------------------------------------------------------------------------------
+# Copyright (c) 2010-2011 zsh-syntax-highlighting contributors
```

Comment:
Please update the copyright years.  You should at least add 2016.  Whether you should retain the older years depends on whether the code was written from scratch or is derived from the `pattern` highlighter's code.